lib/xcres/builder/resources_builder.rb

```diff
@@ -13,12 +13,29 @@ class XCRes::ResourcesBuilder < XCRes::FileBuilder
 //
 EOS
 
-  COMPILER_KEYWORDS = %w{
+  OBJC_COMPILER_KEYWORDS = %w{
     auto break case char const continue default do double else enum extern float
     for goto if inline int long register restrict return short signed sizeof
     static struct switch typedef union unsigned void volatile while
   }
 
+  SWIFT_COMPILER_KEYWORDS = %w{
+    class deinit enum extension func import init internal let operator private
+    protocol public static struct subscript typealias var break case continue
+    default do else fallthrough for if in return switch where while as
+    dynamicType false is nil self Self super true
+  }
+
+  SWIFT_EXTENSIONS = <<EOS
+public extension %s.Strings {
+    public var localizedValue: String {
+        return NSLocalizedString(self.rawValue,
+                                 bundle: NSBundle(forClass: LOC.self),
```

Comment:
Oops 😳 Good catch.
I was testing this on the side in another project, that already had been integrated with xcres. Thus I needed to rename R into something else.

Will fix this in a minute.